amethyst_assets/src/formats.rs

```diff
@@ -23,3 +23,25 @@ where
         Ok(val)
     }
 }
+
+/// Format for loading from Json files.
+#[derive(Default, Clone, Debug)]
+pub struct JsonFormat;
+
+impl<T> SimpleFormat<T> for JsonFormat
+where
+    T: Asset,
+    T::Data: for<'a> Deserialize<'a> + Send + Sync + 'static,
+{
+    const NAME: &'static str = "Json";
+    type Options = ();
+
+    fn import(&self, bytes: Vec<u8>, _: ()) -> Result<T::Data, Error> {
+        use serde_json::de::Deserializer;
+        let mut d = Deserializer::from_slice(&bytes);
+        let val = T::Data::deserialize(&mut d).chain_err(|| "Failed parsing Json file")?;
```

Comment:
Similar to how its done above for Ron, I would say `Failed deserializing JSON file` so that there are not two places that the same message can be produced.

Reply:
I agree with you that that would be usefull, but Ron also has the same error message in these positions: "Failed deserializing Ron file". Changed it